e2e/test_helpers_test.go

```diff
@@ -0,0 +1,128 @@
+package e2e
```

Comment:
seems that  this file is windows specific, should have a better name ?

Reply:
This was a nightmare choosing the name.

We had no tests of functions in a file called "scenario_helpers_test.go", and I really didn't want to create a file called "scenario_helpers_test_test.go" for tests for functions in that file.

What I've done now is:
* extract testable functions from scenario_helpers_test.go into a file "test_helpers.go", and now the file "test_helpers_test.go" is more obviously tests of functions in that file.